src/main/kotlin/uk/gov/gdx/datashare/services/PrisonerApiService.kt

```diff
@@ -0,0 +1,41 @@
+package uk.gov.gdx.datashare.services
+
+import com.amazonaws.xray.spring.aop.XRayEnabled
+import io.micrometer.core.instrument.Counter
+import io.micrometer.core.instrument.MeterRegistry
+import org.springframework.http.HttpStatus
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.reactive.function.client.WebClientResponseException
+import uk.gov.gdx.datashare.config.NoDataFoundException
+import uk.gov.gdx.datashare.models.PrisonerRecord
+
+@Service
+@XRayEnabled
+class PrisonerApiService(
+  private val prisonerSearchApiWebClient: WebClient,
+  meterRegistry: MeterRegistry,
+) {
+  private val callsToPrisonerSearchCounter: Counter = meterRegistry.counter("API_CALLS.CallsToPrisonerSearch")
```

Comment:
Nothing to do with this, but I think we should change our naming scheme for metrics